### DISCUSSION TITLE
Imaginary Maintainer / Nextcloud Fork

### DISCUSSION THREAD

**Discussion by armouredking** [nextcloud/all-in-one#1908] [Questions] [answered]
Not sure if this is the right forum for this discussion, but wanted to ask about it.

Imaginary's original maintainer has stated they cannot provide support & maintenance of the project reliably, and you can tell in the past few years how things have slowed down [#128](https://github.com/h2non/imaginary/issues/128).

Nextcloud appears to be moving towards a further dependence on Imaginary, which is great as it seemingly does a much better job (particularly with security) than ImageMagick. But I'm curious, is Nextcloud willing to advance / maintain it, or is it limited to only what has been done? It seems to me that pieces of it are being maintained in other forks (such as [#390](https://github.com/h2non/imaginary/issues/390) and NC maintaining their own docker image) but it appears to be fracturing.

**Comment by szaimen** [comment]
Hi, in my opinion, we already have too many projects to maintain so adding yet another is not a good idea, imho.

But lets seee what @AndyScherzinger says.

**Comment by AndyScherzinger** [comment, marked as answer]
Currently we are not planning to take over the maintenance of imaginary but only offer the integration into Nextcloud that is currently already provided by and within Nextcloud.
